external/mbedtls/Make.defs

```diff
@@ -53,4 +53,7 @@
 ifeq ($(CONFIG_NET_SECURITY_TLS),y)
 CONFIGURED_EXT += mbedtls
 endif
+ifeq ($(CONFIG_TC_SFS),y)
```

Comment:
What if both CONFIG_NET_SECURITY_TLS and CONFIG_TC_SFS is enabled? It will added two times right.

Reply:
Tried using "or" command. But still facing build errors.